### DISCUSSION TITLE
forecast-v3.weather.gov

### DISCUSSION THREAD

**Discussion by branmh** [weather-gov/api#392] [Q&A]
Hello, 

I'm curious about what happened to forecast-v3.weather.gov? I'm getting unable to find the server. 

https://forecast-v3.weather.gov/products/locations/DY1/FWD/1?format=text

**Comment by kollymars** [comment]
Please see https://www.weather.gov/media/notification/scn16-35forecast_govaah.pdf

The public facing web site has been discontinued until further notice, however the API endpoints remain active.
